Doc/includes/sqlite3/createdb.py

```diff
@@ -12,15 +12,15 @@
 con = sqlite3.connect(DB_FILE)
 cur = con.cursor()
 cur.execute("""
-        create table people
+        create table lang
         (
-          name_last      varchar(20),
-          age            integer
+          name           varchar(20),
+          first_appeared integer
         )
         """)
 
-cur.execute("insert into people (name_last, age) values ('Yeltsin',   72)")
-cur.execute("insert into people (name_last, age) values ('Putin',     51)")
+cur.execute("insert into lang (name, first_appeared) values ('Forth', 1970)")
```

Comment:
Probably only the fourth time in my life I've heard of Forth 😉.

Reply:
Forth is a very interesting language! The bootloader on older Apple computers used to run a kind of a Forth interpreter. Great fun :)

Reply:
https://en.wikipedia.org/wiki/Open_Firmware